### ISSUE TITLE
", } { }, { , bugs.

### ISSUE THREAD

**Issue by turenc** [webpro/knip#418]
Hello,

First off, kudos to the entire team for the fantastic job on this project! 
Impressive Work! 🚀

I'd like to bring an issue to your attention, Here are the details:

Case:

export { RangeSlider } from "./range-slider/range-slider";
export { Rating } from "./rating/rating";
export { Col, Col as KCol } from "./responsive/col";
export { Row as KRow, Row } from "./responsive/row";
export { MenuItem } from "./sider-menu/menu-item";
export { Video as KVideo, Video } from "./video/video";
export { Panel as KPanel, Panel } from "./panel/panel";

Command:  > knip --fix

Result:

export {  } from "./range-slider/range-slider";
export { Rating } from "./rating/rating";
export { , Col as KCol } from "./responsive/col";
export { Row as KRow,  } from "./responsive/row";
export {  } from "./sider-menu/menu-item";
export { ,  } from "./video/video";
export { Panel as KPanel,  } from "./panel/panel";

Expected Behavior:
There should be no ", } { }, { , etc. 

Thank you for your time and for maintaining such a remarkable project!

Best regards,
Türenç.

**Comment by turenc** [comment]
workaround:

# script.sh

while IFS= read -r line; do
  # If the line does not contain "export { , } from", "export { , , } from", "export { , , , } from", or similar expressions, print the line
  if [[ ! $line =~ export\ \{[[:space:],]*\}[[:space:]]*from ]]; then
    # Remove comma characters and print the line
    echo "${line//,/}"
  fi
done < src/components/web-components/index.ts > temp_file

# Replace the original file with the temporary file
mv temp_file src/components/web-components/index.ts

**Comment by webpro** [comment]
Yes, this is noted here: https://knip.dev/features/auto-fix#notes

Please use [code blocks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) to make code more readable.

Thanks for adding the workaround! That's useful in the meantime.

**Comment by turenc** [comment]
Thnx :) 

also with webpack changes as below and it works fine on windows, mac or linux.

```js

const WebpackShellPluginNext = require("webpack-shell-plugin-next");

const config = {
  onBeforeBuild: {
    scripts: [
      "prettier --write --check src/screens",
      "knip --fix",
      "bash knip-clr.sh",
      "prettier --write --check src/components/web-components/index.ts",
      'echo "                                                                            "',
      'echo "     ...Please wait!                                                  "',
      'echo "     ...Building your app...                                      "',
      'echo "                                                                            "'
    ],
    blocking: true
  }
};

```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.2.0. See [Release 5.2.0](https://github.com/webpro/knip/releases/tag/5.2.0) for release notes._
